fixed-price-subscriptions/client/react/vite.config.mjs

```diff
@@ -6,6 +6,8 @@ export default defineConfig({
     react(),
   ],
   server: {
+    host: '0.0.0.0',
+    port: 3000,
```

Comment:
@hideokamoto-stripe This changes Vite's port from `5173` to `3000`. I did it because the current CI configuration expects the port `3000` for frontend servers. Also, I think keep the port unchanged from the original `3000` less confuse the users. But let me know if we should keep the default port number 🙏